File: 6-kyu/parabolic-arc-length/javascript/solution.js
function lenCurve(n) {
  const h = 1/n;
  const y = x => x * x;
  let len = 0;
  for (let i = 0; i < n; i++) {
      let xi = i * h;
      let xf = (i + 1) * h;
      len += Math.sqrt(Math.pow(y(xf) - y(xi), 2) + Math.pow(xf - xi, 2), 2);
      len = Math.round(len * 1e9)/1e9;
  }
  return len
}

File: 6-kyu/parabolic-arc-length/python/solution.py
def len_curve(n):
    h = 1/n
    y = lambda x: x**2
    length = 0
    for i in range(n):
        xi, xf = i * h, (i + 1) * h
        length += ((y(xf) - y(xi))**2 + (xf - xi)**2)**.5
    return round(length, 9)